apps/qubit/modules/clipboard/actions/viewAction.class.php

```diff
@@ -66,6 +66,11 @@ public function execute($request)
             'repository' => sfConfig::get('app_ui_label_repository'),
         ];
 
+        // Some users cannot see accessions - don't show them in that case
+        if ($this->context->user->hasCredential(['contributor', 'editor', 'administrator'], false)) {
```

Comment:
same issue as mentioned earlier, only those with accession edit permissions should be able to export accesions